kANe/v4/2_antyeShTi_Ashaucha_shuddhi/07_Eschatology.md

```diff
@@ -362,7 +359,8 @@ who had not kept sacred firas. 497 The Pār, gr, further provides that all relat
 
 
 
-[.Vol. beat their right thighs, fan the corpse with the fringes of their garments and go thrice round the corpse with the left side towards the corpse and recite the mantra 'apa nah sośncad agham' (Ṛg. I 97 1. Tai. Ā, VI 10.1). It further provides (28, 1 37-46) that the corpse is carried in a cart or by (four) men and while being carried four stops are made, at each of which clods of earth are dug up and boiled rice is offered thereon with mantras like 'pūsā tretah' (Ṛg. X. 17 3., Tai Ā VI, 10 1), Sayur vispāyur" (Ṛg. X. 17. 4, Tai. Ā VI 10.2). The Varāha purāna prescribes that a Paurānic mantra should be recited, that the performer should go round the pyre and apply the kindled fire towards the part of the pyre where the head is placed. 499 ; In modern times the procedure of cremation is generally on the lines of that described in the Asy. gr, cited above and the procedure described in the Garuḍapurāpa II, 4, 41 ff. The following is a summary of the procedure set out in the Antyesti paddhati of Narāyanabhatta which is generally followed by Ṛgveding in Western India. The dead body is washed with water, is decked with a talaka on the forehead, is clothed in a garment, is covered with a paste of Jatāmāmsi and a garland of naladas is placed round its neck, Then the performer washes his own fest, sips water, performs prādāyāma without Vedic mantra, then makes a Saṅkalpa (as in note 500 ),500 he wears his sacred thread under the left shoulder (and on the right shoulder) and prays to the earth with the mantra 'upasarpa' (Ṛg. X. 18.10), he then thrice goes round the place dug up (for cremation) and sprinkles water with a sami branch at the end of the mantra
+
+In modern times the procedure of cremation is generally on the lines of that described in the Asy. gr, cited above and the procedure described in the Garuḍapurāpa II, 4, 41 ff. The following is a summary of the procedure set out in the Antyesti paddhati of Narāyanabhatta which is generally followed by Ṛgveding in Western India. The dead body is washed with water, is decked with a talaka on the forehead, is clothed in a garment, is covered with a paste of Jatāmāmsi and a garland of naladas is placed round its neck, Then the performer washes his own fest, sips water, performs prādāyāma without Vedic mantra, then makes a Saṅkalpa (as in note 500 ),500 he wears his sacred thread under the left shoulder (and on the right shoulder) and prays to the earth with the mantra 'upasarpa' (Ṛg. X. 18.10), he then thrice goes round the place dug up (for cremation) and sprinkles water with a sami branch at the end of the mantra
```

Comment:
tilaka on the forehead, feet etc.

Reply:
This paragraph is not proof-read yet. I closed with footnote 493.